stacks/api/function/enqueuer/test/database.spec.ts

```diff
@@ -22,7 +22,7 @@ describe("DatabaseEnqueuer", () => {
     noopTarget.handler = "default";
 
     eventQueue = jasmine.createSpyObj("eventQueue", ["enqueue"]);
-    databaseQueue = jasmine.createSpyObj("httpQueue", ["enqueue"]);
+    databaseQueue = jasmine.createSpyObj("databaseQueue", ["enqueue"]);
```

Comment:
@thesayyn What's this change?

Reply:
I've changed name of the spy cause it was written wrong.

Reply:
@thesayyn Ok so it's a fix of a previous implementation and not relevant with the firehose integration, right?

Reply:
Actually, it is just a name change. It does not affect anything at all. 
It's not relevant to the firehose implementation.

Reply:
I just wanted to change it to prevent misunderstanding.